console/console-init/ui/src/modules/project/ProjectPage.tsx

```diff
@@ -202,6 +254,27 @@ export default function ProjectPage() {
     return true;
   };
 
+  const isEnableAllOptionDisabled = () => {
```

Comment:
we don't need separate function `isEnableAllOptionDisabled` `isDisableAllOptionDisabled`  for enable and disable status, create single function and hanlde based on the function param isEnabled.

 const getOptionStatus = (isEnabled:boolean) => {
    if (selectedProjects && selectedProjects.length > 0) {
      const filteredProjects = selectedProjects.filter(
        project => project.projectType === ProjectTypes.IOT && (isEnabled ? !project.isEnabled:project.isEnabled)
      );
      if (filteredProjects.length === selectedProjects.length) return false;
    }
    return true;
  };

Modify function logic based on the requirement. I jsut added condition for reference.